frontend/src/components/NotifyMemberComponent.vue

```diff
@@ -16,9 +15,13 @@ export default Vue.extend({
   },
   methods: {
     showToast(message) {
-      if (message.type == "ADMIN_LEFT") {
+      if (message.type === "ADMIN_LEFT") {
```

Comment:
probably you could store the current type and update toast only if the message.type differs from current. This could prevent useless updates.

Reply:
If I remember it correctly the only message type that can be send repeatedly is _MEMBER_LEFT_. So there it would make sense, but in my opinion every member should get notified every time when another member leaves the session (but it is not used here).
**TL:DR** _ADMIN_LEFT_ and _ADMIN_JOINED_ do not get send successively.